CONTRIBUTING.md

```diff
@@ -33,29 +33,35 @@ cargo clippy --workspace --tests -- --deny warnings
 
 Execute `cargo fmt --all` to fix the formatting.
 
-## Generating Docs
-
-If you make changes to [Roc's Standard Library](https://www.roc-lang.org/builtins/Str), you can add comments to the code following [the CommonMark Spec](https://spec.commonmark.org/current/) to further explain your intentions. You can view these changes locally with:
-
-```sh
-cargo run docs crates/compiler/builtins/roc/main.roc
-```
-
-This command will generate the documentation in the [`generated-docs`](generated-docs) directory.
-
```

Comment:
Most PRs won't need this, so this feels too niche for this doc.

Reply:
Agree